client/src/components/Order/FormModal.jsx

```diff
@@ -0,0 +1,115 @@
+import { SmallAddIcon } from '@chakra-ui/icons';
+import {
+  Button,
+  Center,
+  FormControl,
+  FormLabel,
+  Input,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  NumberDecrementStepper,
+  NumberIncrementStepper,
+  NumberInput,
+  NumberInputField,
+  NumberInputStepper,
+  Select,
+  Textarea,
+} from '@chakra-ui/react';
+import useOrderForm from 'hooks/useOrderForm.jsx';
+import PropTypes from 'prop-types';
+import React, { Fragment } from 'react';
+
+const FormModal = ({ showButton = false }) => {
+  // TODO: use the endpoint to get this constant list of materials rather than hard coding
+  const materialTypes = ['rubber', 'aluminum', 'steel', 'copper', 'plastic', 'leather'];
```

Comment:
is it better if we put `materialTypes ` in the constants.js file?

Reply:
Yes, at the very least it should be in the `constants.js` but I left a TODO for Tiff to take a look at (same comment from her PR)

Reply:
i believe naasir is adding code to fetch this from the backend, so we could just wait for that